docs/source/bounded_variables.rst

```diff
@@ -0,0 +1,61 @@
+=================
+Bounded Variables
+=================
+
+PyMC3 includes the construct ``Bound`` for placing constraints on existing
+probability distributions.  It modifies a given distribution to take values
+only within a specified interval.  
+
+Some types of variables require constraints.  For instance, it doesn't make
+sense for a standard deviation to have a negative value, so something like a
+Normal prior on a parameter that represents a standard deviation would be
+inappropriate.  PyMC3 includes distributions that have positive support, such
+as ``Gamma`` or ``Exponential``.  PyMC3 also includes several bounded
+distributions, such as ``Uniform``, ``HalfNormal``, and ``HalfCauchy``, that
+are restricted to a specific domain.  
+
+All univariate distributions in PyMC3 can be given bounds.  The distribution of
+a continuous variable that has been bounded is automatically transformed into
+an unnormalized distribution whose domain is unconstrained.  The transformation
+improves the efficiency of sampling and variational inference algorithms.
+
+Usage
+#####
+
+For example, one may have prior information that suggests that the value of a
+parameter representing a standard deviation is near one.  One could use a
+Normal distribution while constraining the support to be positive.  The
+specification of a bounded distribution should go within the model block::
+
+    import pymc3 as pm
+
+    with pm.Model() as model:
+        BoundedNormal = pm.Bound(pm.Normal, lower=0.0)
+
+    with model:
+        x = BoundedNormal('x', mu=1.0, sd=3.0)
+        
+If the bound will be applied to a single variable in the model, it may be
+cleaner notationally to define both the bound and variable together. ::
+
+    with model:
+        x = pm.Bound(pm.Normal, lower=0.0)('x', mu=1.0, sd=3.0)
+
+Caveats
+#######
+
+* On the master branch, bounds can also be applied to vectors of random
```

Comment:
I think you can do that on the release, too. But on master you can now do something like this:
```
BoundNormal = pm.Bound(pm.Normal, lower=[0, 1])  # The bound is a vector
a = BoundNormal('a', mu=0, sd=1, shape=2)
```
This isn't a caveat, right?